go.mod

```diff
@@ -31,12 +31,13 @@ require (
 	github.com/hashicorp/golang-lru/v2 v2.0.7
 	github.com/hashicorp/raft v1.7.3
 	github.com/hashicorp/raft-boltdb/v2 v2.3.1
+	github.com/hf/nitrite v0.0.0-20241225144000-c2d5d3c4f303
 	github.com/hf/nsm v0.0.0-20220930140112-cd181bd646b9
 	github.com/holiman/uint256 v1.3.2
 	github.com/ipfs/go-datastore v0.6.0
 	github.com/ipfs/go-ds-leveldb v0.5.0
 	github.com/klauspost/compress v1.18.0
-	github.com/kurtosis-tech/kurtosis/api/golang v1.6.0
+	github.com/kurtosis-tech/kurtosis/api/golang v1.9.1
```

Comment:
This kurtosis update is incompatible with the current kurtosis stuff.

Additionally this forces the go version to be `1.23` instead of `1.22`, thereby making the project no longer buildable.

This also applies to other dependencies added here, as with the loosening of the restriction to go `1.22`, other dependencies were able to be moved forward that also require go `1.23`.

I suggest to revert the changes to this file entirely, run

```sh
go mod tidy
```

And ensure that the `go.mod` doesn't update the toolchain / version.  Then check the `go.sum` file to make sure the changes make sense.